### DISCUSSION TITLE
Show a predetermined list of actors

### DISCUSSION THREAD

**Discussion by cisnerosk** [jnunemaker/flipper#732] [Q&A]
Hello there! 

I'm very new to Flipper and I didn't see that this was possible in the documentation, although it's very possible I missed it. I'd like to know if there a way in the Flipper UI to have validation or a drop down of allowed actors. My concern/use case is that if I'm allowing folks who aren't as technical to add actors to a feature, but that actor doesn't exist/they have a typo, they can still add that actor and never realize they didn't add an actor in the first place. If I can add validation or even better yet, a drop down of actors that are available for that feature, that reduces the risk of folks adding actors that don't actually exist.

I hope this question makes sense, and I appreciate your time. Thank you!

**Comment by jnunemaker** [comment]
Great question. I don't think a drop down would be a good idea as that would mean loading thousands to millions of actors just to pick one. Probably slow and not a great experience. But I like the idea of validation that it is a real actor. 

We could probably add a validation block similar to the description block that allows you to validate an actor. If the block returns true the actor is valid else it's not.

Any thoughts on that? The other route would be integrating a block to allow for searching. If there are no results then the actor isn't legit.

**Reply by cisnerosk** [reply]
Ah, great point about the slowness from a drop down perspective, I didn't consider that. That validation idea similar to the description block makes sense to me! I appreciate the response and the thought process and sharing behind it, thank you!

**Reply by jnunemaker** [reply]
Anytime. If you want to take a crack at it, I'm happy to give pointers.
